extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/RouteBuildItem.java

```diff
@@ -43,6 +44,7 @@ public static Builder builder() {
         this.routerType = routerType;
         this.notFoundPageDisplayableEndpoint = builder.getNotFoundEndpoint();
         this.configuredPathInfo = builder.getRouteConfigInfo();
+        this.absolutePath = builder.absolutePath;
```

Comment:
We already built a proper `absolutePath` so let's expose it rather than rebuilding it incorrectly at a later stage.